packages/app/src/server/models/user-registration-order.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-use-before-define */
```

Comment:
>　- Implement as a typescript file
    - Refs: update-post.ts
　- Implement as a scratch code
    - Do not base it on the current User schema
    
by Yuki san